kafka/install.sh

```diff
@@ -41,9 +42,10 @@ cat > /etc/service/kafka/run <<-"EOF"
 #!/bin/sh
 sv start zookeeper || exit 1
 if [[ -z "$KAFKA_ADVERTISED_HOST_NAME" ]]; then
-  echo advertised.listeners=PLAINTEXT://$(route -n | awk '/UG[ \t]/{print $2}'):9092 >> /kafka/config/server.properties
+listeners=PLAINTEXT://:9092
+  echo advertised.listeners=PLAINTEXT://$(route -n | awk '/UG[ \t]/{print $2}'):9092,PLAINTEXT_HOST://localhost:19092 >> /kafka/config/server.properties
```

Comment:
Wondering if we can replace `$(route -n | awk '/UG[ \t]/{print $2}')` for `$HOSTNAME`. The hostname is accessible from other containers in the same network. 

Checking what is produced: 
```
bash-4.4# cat config/server.properties
broker.id=0
zookeeper.connect=127.0.0.1:2181
replica.socket.timeout.ms=1500
log.dirs=/kafka/logs
auto.create.topics.enable=true
offsets.topic.replication.factor=1
listeners=PLAINTEXT://0.0.0.0:9092,PLAINTEXT_HOST://0.0.0.0:19092
listener.security.protocol.map=PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
advertised.listeners=PLAINTEXT://172.18.0.1:9092,PLAINTEXT_HOST://localhost:19092
bash-4.4# cat /etc/hosts
127.0.0.1       localhost
::1     localhost ip6-localhost ip6-loopback
fe00::0 ip6-localnet
ff00::0 ip6-mcastprefix
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters
172.18.0.3      a0957022c248
```

Should not be `advertised.listeners=PLAINTEXT://172.18.0.3:9092,PLAINTEXT_HOST://localhost:19092`?